src/r3_hex_pb_names.erl

```diff
@@ -0,0 +1,735 @@
+%% Vendored from r3_hex_core v0.4.0, do not edit manually
```

Comment:
@starbelly fyi, I've updated vendor.sh on master so that this will be:

```erlang
%% Vendored from hex_core v0.4.0, do not edit manually
```

(drops `r3_` prefix in this particular case)

Reply:
> hex_core v0.4.0

Let us know when this PR is ready to go, I think we should cut a new hex_core version then so this would become v0.5.0. Not a huge deal code wise of course, just more accurate in case someone looks at it.

Reply:
@wojtekmach makes sense to me.... just re-synced. As far as the PR forgoing others who might want to test, I'm good to go. FYI for people... here is a simple shell script to help test publish and fetching private packages : https://gist.github.com/starbelly/e7c0604513e41d0a2af456aa7abcc8a2#the-script